{"name": "Camera Section enabled", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.\nStacktrace:\nNoSuchElementError: An element could not be located on the page using the given search parameters.\n    at AndroidUiautomator2Driver.findElOrEls (C:\\Users\\Dangelo\\.appium\\node_modules\\appium-uiautomator2-driver\\node_modules\\appium-android-driver\\lib\\commands\\find.ts:86:11)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at AndroidUiautomator2Driver.findElOrElsWithProcessing (C:\\Users\\Dangelo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\find.ts:60:12)\n    at AndroidUiautomator2Driver.findElement (C:\\Users\\Dangelo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\find.ts:75:12)\n", "trace": "  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login_steps.py\", line 48, in valid_the_camera_section\n    assert bool(context.app.main_page.valid_capturar_foto())\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\features\\pages\\main_page.py\", line 64, in valid_capturar_foto\n    capturar_foto_button = self.find_element(self.capturar_foto_btn).is_displayed()\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\features\\pages\\base_page.py\", line 11, in find_element\n    return self.driver.find_element(by=locator[0],\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 371, in find_element\n    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 30, in check_response\n    raise wde\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 26, in check_response\n    super().check_response(response)\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given The user input an name", "status": "passed", "start": 1709652233306, "stop": 1709652236607}, {"name": "When Select the Camera section", "status": "passed", "start": 1709652236607, "stop": 1709652237590}, {"name": "And Click on the option while using the app", "status": "passed", "start": 1709652237590, "stop": 1709652238777}, {"name": "Then valid that the camera section is opened", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.\nStacktrace:\nNoSuchElementError: An element could not be located on the page using the given search parameters.\n    at AndroidUiautomator2Driver.findElOrEls (C:\\Users\\Dangelo\\.appium\\node_modules\\appium-uiautomator2-driver\\node_modules\\appium-android-driver\\lib\\commands\\find.ts:86:11)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at AndroidUiautomator2Driver.findElOrElsWithProcessing (C:\\Users\\Dangelo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\find.ts:60:12)\n    at AndroidUiautomator2Driver.findElement (C:\\Users\\Dangelo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\basedriver\\commands\\find.ts:75:12)\n", "trace": "  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login_steps.py\", line 48, in valid_the_camera_section\n    assert bool(context.app.main_page.valid_capturar_foto())\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\features\\pages\\main_page.py\", line 64, in valid_capturar_foto\n    capturar_foto_button = self.find_element(self.capturar_foto_btn).is_displayed()\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\features\\pages\\base_page.py\", line 11, in find_element\n    return self.driver.find_element(by=locator[0],\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 371, in find_element\n    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 30, in check_response\n    raise wde\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 26, in check_response\n    super().check_response(response)\n  File \"E:\\trabajo\\Andina\\appium-python-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1709652238777, "stop": 1709652299343}], "start": 1709652229773, "stop": 1709652299485, "uuid": "bba31633-efc9-4d81-adca-44269c2a54e8", "historyId": "92e87052ca7fd142baea1b27613ca7f7", "fullName": "Tests for MyWorkDoc application: Camera Section enabled", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Tests for MyWorkDoc application"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}